packages/@azure/servicebus/data-plane/examples/javascript/gettingStarted/sendWithInteractiveAuth.js

```diff
@@ -0,0 +1,56 @@
+/*
+  This sample demonstrates how to create a namespace using TokenCredentials.
+
+  Setup: AAD RBAC is enabled only on the new namespaces in these regions
+    - [US East, US East 2, or West Europe] for the preview.
+    Please ensure that your servicebus-namespace is present in one of these regions.
+*/
+
+const { Namespace } = require("@azure/service-bus");
+const { interactiveLogin } = require("ms-rest-azure");
+
+const aadServiceBusAudience = "https://servicebus.azure.net/";
+// Define Service Bus Endpoint here and related entity names here
+const serviceBusEndpoint = ""; // <your-servicebus-namespace>.servicebus.windows.net
+const queueName = "";
+
+const listOfScientists = [
+  { name: "Einstein", firstName: "Albert" },
+  { name: "Heisenberg", firstName: "Werner" },
+  { name: "Curie", firstName: "Marie" }
+];
+
+async function main(){
+  // tokenCreds can be of the following types
+  //      ApplicationTokenCredentials
+  //      UserTokenCredentials
+  //      DeviceTokenCredentials
+  //      MSITokenCredentials
+  // interactiveLogin returns DeviceTokenCredentials
```

Comment:
Do we really need the above comments?
The intellisense on `interactiveLogin` and `createFromAadTokenCredentials` already talk about the things noted above

Reply:
Same for the other file

Reply:
Initially, those comments made sense when I was planning to add only one sample for different types of tokenCreds.


Will remove that now.